PrivateCollections/Ethiopia/EsatZammana/ESZA001.xml

```diff
@@ -0,0 +1,327 @@
+                                    <ab type="ruling">The upper line is written above the ruling,
+                                        the lower line is written above the ruling.</ab>
+                                </layout>
+                            </layoutDesc>
+                           </objectDesc>
+                         <handDesc>
+                             <handNote xml:id="h1" script="Ethiopic">
+                                 <desc>Regular hand.The manuscript was copied by <persName ref="PRS14593EsatZammana"/>.
+                                 </desc>
+                                 <seg type="ink">Black, red.</seg>
+                                 <date from="2020" to="2022"/>                                 
+                             </handNote>                   
+                         </handDesc>                                            
+                        <bindingDesc>
+                            <binding contemporary="true" xml:id="binding">
+                                <decoNote xml:id="b1">In the original boards, lined with flowered silk and covered with blind-tooled leather.</decoNote>
+                                <decoNote xml:id="b2" type="bindingMaterial">
+                                    <material key="wood"/>
+                                    <material key="leather"/>
+                                    <material key="textile"/>
+                                </decoNote>
+                            </binding>
+                        </bindingDesc>
+                     </physDesc>               
+                       <history>
+                        <origin>
+                            <origDate notBefore="2020" notAfter="2022" evidence="context"/>
+                        </origin>
+                    </history>             
+                 </msDesc>
+            </sourceDesc>
+
+        </fileDesc>
+        <encodingDesc>
+            <projectDesc>
+                <p>Encoded according to TEI P5 Guidelines.</p>
+            </projectDesc>
+        <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+        <profileDesc>
+            <textClass>
+                <keywords>                   
+                    <term key="Miracle"/>
+                    <term key="ChristianLiterature"/>                    
+                </keywords>
+            </textClass>
+            <langUsage><language ident="gez">Gǝʿǝz</language>
+             <language ident="en">English</language></langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="AY" when="2024-12-16">Created record</change>            
+        </revisionDesc>
+    </teiHeader>
+    <facsimile>
```

Comment:
It would have been better if you waited for @thea-m to review. We should always try to have 2 reviews per pull request.
If you are too much in a hurry then please make sure that when using an old record as a basis for a new one you always remove what does not belong. (I already did it for this record now)